Source/WebCore/platform/MIMETypeRegistry.cpp

```diff
@@ -411,6 +411,7 @@ bool MIMETypeRegistry::isSupportedImageMIMEType(const String& mimeType)
 {
     if (mimeType.isEmpty())
         return false;
+    auto normalizedMIMEType = normalizedImageMIMEType(mimeType);
```

Comment:
Do we need to use normalizedImageMIMEType() in isSupportedImageMIMETypeForEncoding() also? Should we implement isJPEGMIMEType() using normalizedImageMIMEType()?